### PR TITLE
Gem display - fixes #4645

### LINKED ISSUE #4645: Gems display in item purchase popup is a little confusing
Today I bought a Fortify Potion, and was disoriented for a minute because the upper right corner of the modal said 5 Gems, but the price of the item was 4 Gems. I thought there was some sort of inconsistency going on! After my double-take, I realized that the 5 was my current balance, not the price of the item. Perhaps the Gem balance readout could say "You have N Gems" instead of simply "N Gems", for clarity?

### PR DESCRIPTION
**Pull request by remmz0r**
Fixes #4645 

Sorry about the crazy merges in the branch (yet again). 

Fixes gem display to say "[Gem] You have [n] Gems"

fortify
buygems

### CODE REVIEW

**Comment by crookedneighbor** [comment]
@remmz0r Can you attach a screenshot?

**Comment by remmz0r** [comment]
@crookedneighbor - sorted mate, I can edit it to be "You have [gem] [n] Gems" too if you'd prefer that.

Again, I think I accidentally committed the changelog too -_- Which is why there's so many lines changed in this branch.

File: common/locales/en/generic.json [hunk 1]
```diff
@@ -38,6 +38,7 @@
     "alreadyHave": "Whoops! You already have this item. No need to buy it again!",
     "delete": "Delete",
     "gems": "Gems",
+    "startGems": "You have ",
```
Comment (crookedneighbor) [inline common/locales/en/generic.json]: To make this better for translators, I'd actually like to see this be:

```
"gemButton": "You have <%= number %> Gems.",
```

[Here's an example of how to use a translatable string variable in a jade file](https://github.com/HabitRPG/habitrpg/blob/develop/website/views/shared/header/menu.jade#L198)

**Comment by remmz0r** [comment]
@crookedneighbor - I think this might be to your liking! Always learning :+1:

**Comment by chimericdream** [comment]
@remmz0r I like this change, but don't like that there is an extraneous commit with all of the changelog stuff. Could you cherry-pick those changes out of the PR (or make another one if that's quicker since the changes aren't too numerous)?

**Comment by remmz0r** [comment]
@chimericdream - Fixing this now

**Pull request closed without merging**